### PR TITLE
Added checks from D7 module.

### PR DESCRIPTION
**Pull request by benjy**

### CODE REVIEW

File: src/Checker/ModuleStatusChecker.php [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+   * @return string
+   *   The check message.
+   */
+  protected function getDescription($module) {
+    $status = $module['status'];
+
+    switch ($status) {
+      case UPDATE_CURRENT:
+        $message = $this->t('Up to date (existing_version)', [
+          $module['existing_version'],
```
Comment (kimpepper) [inline src/Checker/ModuleStatusChecker.php]: I thought we still used `@` for variables?

File: src/Checker/PerformanceChecker.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    parent::__construct($string_translation);
+    $this->configFactory = $config_factory;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getChecks() {
+    $checks = [];
+    $config = $this->configFactory->get('system.performance');
```
Comment (kimpepper) [inline src/Checker/PerformanceChecker.php]: I would normally do this in the constructor and store $this->config

File: tests/src/Unit/PerformanceCheckerTest.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+namespace Drupal\Tests\dashboard_connector\Unit;
+
+use Drupal\dashboard_connector\Checker\PerformanceChecker;
+use Prophecy\Argument;
+
+/**
+ * Tests the performance checker plugin.
+ */
+class PerformanceCheckerTest extends \PHPUnit_Framework_TestCase {
```
Comment (kimpepper) [inline tests/src/Unit/PerformanceCheckerTest.php]: If we extend from core's UnitTestBase we get a mock configfactory helper.

File: src/Checker/ModuleStatusChecker.php (hunk 1 above)
Comment (benjy) [inline src/Checker/ModuleStatusChecker.php]: Only if you want them escaped.

File: src/Checker/PerformanceChecker.php (hunk 2 above)
Comment (benjy) [inline src/Checker/PerformanceChecker.php]: Yeah we can do that but it clutters the amount of properties you need if you have multiple config objects in use.

File: tests/src/Unit/PerformanceCheckerTest.php (hunk 3 above)
Comment (kimpepper) [inline tests/src/Unit/PerformanceCheckerTest.php]: Will these tests be able to be run on d.o?

**Pull request merged**